src/app/homepage/registration/registration.component.html

```diff
@@ -1,6 +1,11 @@
 <form class="form-container">
   <h2 mat-dialog-title>Registration</h2>
   <div mat-dialog-content class="dialog-container">
+    <mat-form-field>
+      <label>
```

Comment:
I think this label is not a recommended way by angular material.

Reply:
It's not recommended by Angular material, but I got warning from Angular that it's needed!